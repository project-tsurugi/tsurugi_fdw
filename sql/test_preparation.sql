 -- error
CREATE TABLE tsurugifdw_regressiontest_table(id INTEGER);
CREATE EXTENSION tsurugi_fdw;
DROP TABLE tsurugifdw_regressiontest_table;

CREATE EXTENSION IF NOT EXISTS tsurugi_fdw;
CREATE SERVER IF NOT EXISTS tsurugidb FOREIGN DATA WRAPPER tsurugi_fdw;

\dx tsurugi_fdw
