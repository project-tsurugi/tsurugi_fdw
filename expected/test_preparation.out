CREATE EXTENSION IF NOT EXISTS tsurugi_fdw;
CREATE SERVER IF NOT EXISTS tsurugidb FOREIGN DATA WRAPPER tsurugi_fdw;
\dx tsurugi_fdw
                   List of installed extensions
    Name     | Version | Schema |           Description            
-------------+---------+--------+----------------------------------
 tsurugi_fdw | 1.1.1   | public | foreign-data wrapper for tsurugi
(1 row)

